index.html

```diff
@@ -1,167 +1,276 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Raj</title>
 
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,400;0,600;0,700;1,100;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,400;0,600;0,700;1,100;1,300;1,400;1,600;1,700&display=swap"
+      rel="stylesheet"
+    />
 
-    <link rel="stylesheet" href="css/style.css">
-    
-</head>
-<body>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
     <header>
-        <nav class="menu">
-            <div>
-                <a href="#" class="logo">My Logo</a>
-            </div>
-            <div class="menu-icon">
-                <img src="/img/Icon - Menu.png" alt="">
-            </div>
-        </nav>
+      <nav class="menu">
+        <div>
+          <a href="#" class="logo">My Logo</a>
+        </div>
+        <div class="menu-icon">
+          <img src="/img/Icon - Menu.png" alt="" />
+        </div>
+      </nav>
     </header>
 
     <main>
-        <div class="container">
-            <div class="content">
-                <h1 class="my-name">I’m Mariane <br> Glad to see <br> you!</h1>
-                <p class="my-name-sub">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                <a href="#" class="connect-me">LET'S CONNECT</a>
-            </div>
-            <ul class="social-media-icon">
-                <li><img src="/img/social icons/twiter_icon.svg" alt=""></li>
-                <li><img src="/img/social icons/github_icon.svg" alt=""></li>
-                <li><img src="/img/social icons/icon.svg" alt=""></li>
-                <li><img src="/img/social icons/Linkedin_icon.svg" alt=""></li>
-                <li><img src="/img/social icons/medium_icon.svg" alt=""></li>
-            </ul>
```

Comment:
- [ ] As the last reviewer said:

> **Kindly note that these icons are not displaying. Your UI needs to match the Figma design, and these icons need to be visible for when a user of your app may want to click on them**

| **Figma design** | ** Your design** |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/192775207-5a12ccf2-0826-4548-b664-386d20a3d0ef.png) | ![image](https://user-images.githubusercontent.com/58642949/192775341-8366f8de-2553-46ff-ba97-746e4f8fb279.png) |

- [ ] **[OPTIONAL]** Please 🙏🏽 add the background image and the hamburger button in the navbar.

Reply:
![image](https://user-images.githubusercontent.com/107557899/192782794-1e2f567e-8a29-43a0-8719-ddbbb467c0e9.png)
It runs fine on my pc I have attached the image please suggest me